Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -210,6 +210,11 @@ private func pledgeCTA(project: Project, backing: Backing?) -> PledgeStateCTATyp
       return PledgeStateCTAType.viewYourRewards
     }
 
+    if featurePostCampaignPledgeEnabled(), project.postCampaignPledgingEnabled,
+      project.isInPostCampaignPledgingPhase {
```

Comment:
same here; I think we only need to check if it's in the post campaign pledging phase and if the flag is flipped.